iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -385,6 +403,8 @@ public static void checkLossFactor(Validable validable, float lossFactor) {
             throw new ValidationException(validable, "loss factor is invalid");
         } else if (lossFactor < 0) {
             throw new ValidationException(validable, "loss factor must be >= 0");
+        } else if (lossFactor > 3) {
```

Comment:
The lossFactor is defined and calculated as a percentage, so `0 <= lossFactor <= 100`

Reply:
In fact, I ask @mathbagu and @geofjamg and I have checked in some private parts and the ````lossFactor````is not a percentage, it is a ratio. So:
- What is the hupper limit for you ? 1 or more (a ratio could be greater than 1, as a percentage can be greater than 100) ?
- You have to fix your PR about the CGMES conversion of HVDC (sorry for that) because the web site documentation was wrong.
-

Reply:
According with the master version the lossFactor is calculated and expressed as a percentage and to be coherent with the code implementation should be a number in the closed interval [0,100]. (See code lines 179 - 186 and 41 - 93 of the  `DcLineSegmentConversion.java` class). 

It is a valid way to express the loss factor and in the IIDM model there are other attributes expressed in the same way, e.g. the step correction in `ratioTapChanger` and `phaseTapChanger` but we are in the IIDM model so RTE can choose a different way to express it.

Reply:
In fact, the DCLineSegmentConversion.Java has been modified with percentage last year by Miora and me. Miora and me have been influenced by the documentation. You are right about RTC and PTC.